tests/scripts/thread-cert/backbone/test_bmlr.py

```diff
@@ -106,9 +106,13 @@ def test(self):
         self.nodes[ROUTER1].start()
         self.wait_node_state(ROUTER1, 'router', 5)
 
+        self.wait_link_established(ROUTER1, PBBR)
+
         self.nodes[COMMISSIONER].start()
         self.wait_node_state(COMMISSIONER, 'router', 5)
 
+        self.wait_link_established(COMMISSIONER, ROUTER1)
```

Comment:
Commissioner sends petition to Leader, the direct neighbor link should be established soon, while it takes some time for route establishment between Leader and the new router multiple hops away.

Better to verify the route to the Leader?

```suggestion
        self.wait_link_established(COMMISSIONER, PBBR)
```

Reply:
Changed `wait_link_established` to `wait_route_established`